### ISSUE TITLE
Some tools you may like, much more efficient

### ISSUE THREAD

**Issue by Krutonium** [pplude/TronScriptEvo#17]
I wrote a set of tools, RemoveByName and RemoveByGUID that lets you specify a list of applications to remove, and it will remove them for you. It uses WMI, but it does it intelligently - only running the commands it needs to - So your not running removals on software that isn't installed. You interested?

**Comment by pplude** [comment]
Definitely. I want to use WMI as much as possible. It's insanely useful. Someone mentioned that if WMI is broken it would stop the script from running, but if that's the case, there's more of a reason to start over with a new install of Windows. 

So in the interest of not being too long winded, please feel free to rewrite Stage 2. Thank you.

**Comment by Krutonium** [comment]
If WMI is broken, that actually indicates an issue with the PATH variable. In any case, I am not sure if my application be affected, because I am using direct calls to the system to do the work, not the WMI binary.
